buffered_test.cpp

```diff
@@ -0,0 +1,167 @@
+#include <cstdlib>
```

Comment:
You should run the test under a dry run too -- call vlsv::Writer::startDryRun() before vlsv::Writer::open(...), and call vlsv::Writer::endDryRunning() before vlsv::Writer is destructed.